Dockerfile

```diff
@@ -3,6 +3,10 @@ FROM registry.fedoraproject.org/fedora:29
 LABEL maintainer="Factory 2 Team" \
       description="A microservice triggered by specific message to tag a build."
 
+# Required if running in Fedora with fedmsg to publish message.
+# Check this again after migrating to fedora-messaging. If it is not required, remove it.
+ENV USER=mts
```

Comment:
Instead of hard coding a user, can we instead set this environment variable in the `DeploymentConfig`? That way we only have to do this where it's needed, in the Fedora deployment.

Reply:
@lcarva I did it, https://os.stg.fedoraproject.org/console/project/message-tagging-service/browse/dc/mts?tab=environment and https://infrastructure.fedoraproject.org/cgit/ansible.git/tree/roles/openshift-apps/message-tagging-service/templates/deployment.yml#n22. But, USER is not injected into container, it is not listed in the result of command `export` and Python `os.environ` dict.

Reply:
Hmm odd!

I don't have access to the Openshift cluster but your word is good enough for me.